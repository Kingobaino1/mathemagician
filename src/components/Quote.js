import React from 'react';
import Nav from './Nav';

const Quote = () => {
  const quote = 'Famous Quotes';
  return (
    <>
      <Nav />
      <h1 className="w-25 mx-auto">{quote}</h1>
      <div className="row wide">
        <div className="col-3 m-3 pl-5">
          <p>Give me a place to stand, and I will move the earth</p>
          <p className="text-secondary font-weight-light">Archimedes</p>
        </div>
        <div className="col-3 m-3">
          <p>
            If I have been able to see further, it was only because I
            stood on the shoulders of giants.
          </p>
          <p className="text-secondary font-weight-light">Isaac Newton</p>
        </div>
        <div className="col-3 m-3">
          <p>Mathematics is the queen of science, and arithmetic the queen of mathematics</p>
          <p className="text-secondary font-weight-light">Carl Friedrich Gauss</p>
        </div>
      </div>
    </>
  );
};

export default Quote;
