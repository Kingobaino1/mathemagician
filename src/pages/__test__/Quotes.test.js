import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Quote from '../Quote';

describe('Quote', () => {
  it('should match the snapshot', () => {
    const tree = renderer.create(
      <BrowserRouter>
        <Quote />
      </BrowserRouter>,
    );
    expect(tree).toMatchSnapshot();
  });
});
