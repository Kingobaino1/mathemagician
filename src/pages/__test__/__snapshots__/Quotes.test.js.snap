// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote should match the snapshot 1`] = `
Array [
  <div
    className="d-flex justify-content-between"
  >
    <ul
      className="d-none d-md-block m-4"
    >
      <h1
        className="text-dark font-weight-bold"
      >
        Mathemagician
      </h1>
    </ul>
    <ul
      className="nav m-4"
    >
      <li
        className="nav-item"
      >
        <a
          className="nav-link active text-dark font-weight-bold"
          href="/"
          onClick={[Function]}
        >
          Home
        </a>
      </li>
      <li
        className="nav-item"
      >
        <a
          className="nav-link text-dark font-weight-bolder"
          href="/calculator"
          onClick={[Function]}
        >
          Calculator
        </a>
      </li>
      <li
        className="nav-item"
      >
        <a
          className="nav-link text-dark font-weight-bold"
          href="/quotes"
          onClick={[Function]}
        >
          Quotes
        </a>
      </li>
    </ul>
  </div>,
  <h1
    className="w-25 mx-auto"
  >
    Famous Quotes
  </h1>,
  <div
    className="row wide"
  >
    <div
      className="col-3 m-3 pl-5"
    >
      <p>
        Give me a place to stand, and I will move the earth
      </p>
      <p
        className="text-secondary font-weight-light"
      >
        Archimedes
      </p>
    </div>
    <div
      className="col-3 m-3"
    >
      <p>
        If I have been able to see further, it was only because I stood on the shoulders of giants.
      </p>
      <p
        className="text-secondary font-weight-light"
      >
        Isaac Newton
      </p>
    </div>
    <div
      className="col-3 m-3"
    >
      <p>
        Mathematics is the queen of science, and arithmetic the queen of mathematics
      </p>
      <p
        className="text-secondary font-weight-light"
      >
        Carl Friedrich Gauss
      </p>
    </div>
  </div>,
]
`;
